frontend/src/hooks/useGetMetricsStepsEffect.ts

```diff
@@ -21,6 +21,8 @@ export interface useGetMetricsStepsEffectInterface {
 }
 
 const TIMEOUT = 'timeout';
+const ERROR_STATUS = 'rejected';
```

Comment:
better to put these 2 to `resources.ts`, besides, `fulfilled` and `rejected` are the 2/3(the another one is pending) key words of Promise A+ status. You can name them directly by `FULFILLED` & `REJECTED`.